Analyse_Birds.qmd

```diff
@@ -79,8 +81,24 @@ Import dataset
 data <- read.table("Bird_dataset.txt", header = TRUE, stringsAsFactors = TRUE)
 ```
 
+If the data isn't attached yet, please consider doing so
+
+```{r}
```

Comment:
It is not good practiceto attach data as it can interfere with other analysis if two datasets have the same column name

Reply:
This has been updated, there is no need to attach the dataset anymore and this cell has been removed. Thank you for the review.